woocommerce/woocommerce-gutenberg-products-block
# Migrate storybook entries of QuantitySelector component

This PR migrates stories for the `QuantitySelector` component and adds documentation for its props.

It also reverts a workaround introduced in #5354 in lieu of the official workaround recently published on the Storybook docs (see: https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#emotion11-quasi-compatibility).

Fixes #5367

### Testing

### Manual Testing

How to test the changes in this Pull Request:

1. Run `npm run storybook`
2. Explore the [QuantitySelector stories](http://localhost:6006/?path=/story/woocommerce-blocks-base-components-quantityselector--default). There should be two stories: a default one with the controlled component (i.e. can't set the values from the controls, but only from the action), a second one showing the component in its disabled state.

### User Facing Testing
These are steps for user testing (where "user" is someone interacting with this change that is not editing any code).
